.github/e2e-tests.yml

```diff
@@ -974,19 +974,6 @@ runner-test-matrix:
       E2E_TEST_SELECTED_NETWORK: SIMULATED_1,SIMULATED_2,SIMULATED_3
       E2E_JD_VERSION: 0.6.0
 
-  - id: smoke/ccip/fee_boosting_test.go:*
```

Comment:
Ah actually prematurely approved, you need to add this test to `integration-in-memory-tests.yml` in order for it to run in CI